Library/Homebrew/test/utils/bottles/bottles_spec.rb

```diff
@@ -13,4 +13,25 @@
       end
     end
   end
+
+  describe ".load_tab" do
+    context "when tab_attributes and tabfile are missing" do
+      it "includes runtime_dependencies", :integration_test do
```

Comment:
Are you sure this needs to be an `:integration_test`? Would like to avoid adding new ones whenever possible because they are so slow 🐌

Reply:
I added `integration_test`, because I needed `install_test_formula` and  `setup_test_formula` to perform the tests with the formulas. I'd be open to other ways of testing this.

Reply:
Search for `temporarily_install_bottle` or `when it can't remove a keg` or `eligible_kegs_for_cleanup` for some examples of tests that do installation without an integration test.

Reply:
@MikeMcQuaid sorry for the delay, I have fixed this and made the test a non `:integration_test`